packages/block-editor/src/components/inserter/block-list.js

```diff
@@ -135,6 +135,10 @@ function InserterBlockList( {
 		return filter( filteredItems, { category: 'reusable' } );
 	}, [ filteredItems ] );
 
+	const uncategorizedItems = useMemo( () => {
+		return filter( filteredItems, ( item ) => ! item.category );
```

Comment:
```suggestion
		return filteredItems.filter( ( item ) => ! item.category );
```

Or if `filteredItems` can be `nullish`:

```suggestion
		return filteredItems?.filter( ( item ) => ! item.category );
```

Reply:
I was reluctant at first, since for consistency's sake I was merely following the established form. But I think it's fair enough to do a refactoring pass here, since it's likely not to be done for its own sake, and the pull request isn't too overwhelming as it is.

Updated in 90b6470.